build.sh

```diff
@@ -17,6 +17,6 @@ python3 -m pip install --upgrade pip
 python3 -m pip install -r requirements.txt
 # Dynamically find the Python version directory inside .build-env/lib
 PYTHON_LIB_PATH=$(find .build-env/lib -type d -name "python3.*" -print -quit)
-AHRS_UTILS_PATH="$PYTHON_LIB_PATH/site-packages/ahrs/utils"
-python3 -m PyInstaller --add-data "$AHRS_UTILS_PATH:ahrs/utils" --onefile --hidden-import="googleapiclient" src/main.py
+CV2_UTILS_PATH="$PYTHON_LIB_PATH/site-packages/cv2"
```

Comment:
Pete pointed out that we don't need to use `depthai-sdk` as our dep anymore since we've migrated to `depthai`. This fix is to fix the build process post-migration to `depthai`

Reply:
I tested the new build on RPI4 Linux and executes and is good